2/contrib/openshift/base-plugins.txt

```diff
@@ -1,38 +1,37 @@
 ant:1.11
-blueocean:1.24.4
-bouncycastle-api:2.18
-config-file-provider:3.7.1
-configuration-as-code:1.41
+blueocean:1.24.8
+bouncycastle-api:2.23
+config-file-provider:3.8.1
+configuration-as-code:1.52
 configuration-as-code-groovy:1.1
-credentials-binding:1.23
-credentials:2.3.19
-cloudbees-folder:6.15
-docker-commons:1.16
-git-client:3.2.1
-git:4.7.2
-github:1.33.1
-google-oauth-plugin:1.0.0
-groovy:2.2
-htmlpublisher:1.21
-jira:3.0.17
+credentials-binding:1.27
+credentials:2.5
+cloudbees-folder:6.16
+docker-commons:1.17
+git-client:3.9.0
+git:4.8.3
+github:1.34.0
+google-oauth-plugin:1.0.6
+groovy:2.4
+htmlpublisher:1.25
+jira:3.5
 job-dsl:1.77
-junit:1.30
-kubernetes:1.29.7
-kubernetes-client-api:4.13.3-1
-lockable-resources:2.10
+junit:1.52
+kubernetes:1.30.1
+lockable-resources:2.11
 mapdb-api:1.0.9.0
-matrix-auth:2.6.6
-matrix-project:1.18
-mercurial:2.12
-metrics:4.0.2.6
+matrix-auth:2.6.8
+matrix-project:1.19
+mercurial:2.15
+metrics:4.0.2.8
 openshift-client:1.0.35
 openshift-login:1.0.26
 openshift-pipeline:1.0.57
-openshift-sync:1.0.46
+openshift-sync:1.0.50
```

Comment:
Do we really need to update the openshift-sync version? It is quite broken at 1.0.47+
https://github.com/openshift/jenkins-sync-plugin/issues/392
https://issues.jenkins.io/browse/JENKINS-66468 
(same issue reported in both systems because neither seem to be attracting a fix...)
I suspect updating it here will make a lot more people aware of the problems. Which I suppose will raise its priority and force more investigation and a fix.... Crack on!

Reply:
@max-allan-surevine thank you for pointing these issues out. Given the severity of their impact, I am going to keep openshift-sync on 1.0.46.